pkg/images/error.go

```diff
@@ -0,0 +1,12 @@
+package images
+
+import "github.com/giantswarm/microerror"
+
+var executionFailedError = &microerror.Error{
+	Kind: "executionFailedError",
+}
+
+// IsExecutionFailed asserts executionFailedError.
+func IsExecutionFailed(err error) bool {
+	return microerror.Cause(err) == executionFailedError
+}
```

Comment:
This is the only error w/o matcher. See https://github.com/giantswarm/fmt/blob/master/go/errors.md#matching-errors

```suggestion
```